bots/coniksbot/cmd/run.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"os"
+	"os/signal"
+	"path"
+
+	"github.com/coniks-sys/coniks-go/bots"
+	"github.com/spf13/cobra"
+)
+
+// runCmd represents the run command
+var runCmd = &cobra.Command{
+	Use:   "run",
+	Short: "Run a CONIKS bot instance",
+	Long: `Run a CONIKS bot instance
+
+This will look for config files with default names (botconfig.toml) 
+in the current directory if not specified differently.
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		dir := cmd.Flag("dir").Value.String()
+		config := cmd.Flag("config").Value.String()
+
+		confPath := path.Join(dir, config)
+		run(confPath)
+	},
+}
+
+func init() {
```

Comment:
Can you make this like https://github.com/coniks-sys/coniks-go/blob/master/keyserver/coniksserver/cmd/run.go#L35

with a pid and dir removed.